test/binCases/output/output-library-many/stdin.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+
+module.exports = function testAssertions(code, stdout, stderr) {
+	expect(code).toBe(0);
+	expect(stdout).toEqual(expect.anything());
+	expect(stdout[7]).toMatch(/index\.js.*\{0\}/);
+	expect(stderr).toHaveLength(0);
+
+	const output = fs.readFileSync(path.join(__dirname, "../../../js/bin/output/output-library-many/main.js"), "utf-8");
```

Comment:
could you abstract this a lil better? Quite long line and slashes that could be trimmed down using `path.resolve`